packages/react-portal/src/components/Portal/Portal.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { usePortal } from './usePortal';
+import { PortalProps } from './Portal.types';
+import { renderPortal } from './renderPortal';
+
+/**
+ * Component that renders a slot in a React portal
```

Comment:
```suggestion
 * Component that renders children in a React portal
```

As we don't render slots, is it cleaner?